clients/src/main/java/org/apache/kafka/common/requests/DeleteTopicsRequest.java

```diff
@@ -101,6 +101,12 @@ public AbstractResponse getErrorResponse(int throttleTimeMs, Throwable e) {
             return data.topics().stream().map(topic -> topic.name()).collect(Collectors.toList());
         return data.topicNames(); 
     }
+
+    public int numberOfTopics() {
```

Comment:
Should we add a small unit test or extend an existing unit test to verify this across all versions?

Reply:
Can do!